File: Python/Card.py
from payment import Payment
from datetime import date

class Card(Payment):
    cardBank = str
    endDate = date.today()
    cvv = int

    def __init__(self, id, cardBank,endDate, cvv) -> None:
        super().__init__(id)
        self.cardBank = cardBank
        self.endDate = endDate
        self.cvv = cvv

File: Java/Card.java
import java.util.Date;

public class Card extends Payment{
    Integer numbercard;
    Date enddate;
    Integer cvv;

    public Card(Integer id, Integer numbercard, Date enddate, Integer cvv) {
        super(id);
        this.numbercard = numbercard;
        this.enddate = enddate;
        this.cvv = cvv;
    }
}